internal/cmd/gocdk/biome_add.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2019 The Go Cloud Development Kit Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"context"
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	slashpath "path"
+	"path/filepath"
+	"text/template"
+
+	"golang.org/x/xerrors"
+)
+
+func biomeAdd(ctx context.Context, pctx *processContext, args []string) error {
+	// TODO(clausti) interpolate launcher from one supplied as a flag
+	f := newFlagSet(pctx, "demo")
+	usageMsg := "gocdk biome add BIOME_NAME"
```

Comment:
Remove the `BIOME_NAME` here to avoid it being spilled out into usage error messages for which it doesn't make sense. For instance, line 42 would read:

```
gocdk biome add BIOME_NAME: expected add, got xyzzy
```

as opposed to:

```
gocdk biome add: expected add, got xyzzy
```

```suggestion
	const usageMsg = "gocdk biome add"
```